src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -55,20 +55,19 @@ template<class ActivationFunction>
 void CheckActivationCorrect(const arma::colvec input,
                             const arma::colvec target)
 {
+  // Adding a temporary colvec
```

Comment:
This isn't too useful a comment---do you think maybe the comment can instead say *why* the temporary colvec is needed?